src/context/api/resend.ts

```diff
@@ -0,0 +1,52 @@
+import { resend } from "@upstash/qstash";
+import { ApiCallSettings, BaseWorkflowApi } from "./base";
+import { CallResponse } from "../../types";
+
+type SendEmail = {
+  from: string;
+  to: string;
+  subject: string;
+  bcc?: string | string[];
+  cc?: string | string[];
+  scheduled_at?: string;
+  reply_to?: string | string[];
+  html?: string;
+  text?: string;
+  headers: unknown;
+  attachments: unknown;
+  tags: { name: string; value: string }[];
+};
+type SendEmailResponse = {
+  id: string;
+};
+
+type SendBatchEmail = SendEmail[];
+type SendBatchEmailResponse = {
+  data: SendEmailResponse[];
+};
+
+export class ResendAPI extends BaseWorkflowApi {
+  public async call<
+    TBatch extends boolean = false,
```

Comment:
would the user code be more readable if we did this like `call<'batch', TBoady>` instead of `call<true, Tbody>`, wdyt?

Reply:
since we use `batch: TBatch`, parameter becomes `batch: "batch"`. We could change the parameter name or use `batch: TBatch extends "batch" ? true : false", but I think it's simpler in the current version

Reply:
Yeah, doesn't even matter that much i guess